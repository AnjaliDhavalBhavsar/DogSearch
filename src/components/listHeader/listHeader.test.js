import TestRenderer from "react-test-renderer";
import DogSearch from "../../container/dogSearch";
import DogDataProvider from "../../context";

const data = {
  dogList: [
    {
      imperial: "23 - 25",
      height: "58 - 64",
      id: 9,
      name: "Alaskan Malamute",
      group: "Working",
      lifespan: "12 - 15 years",
      temperament: "Friendly, Affectionate, Devoted, Loyal, Dignified, Playful",
    },
    {
      imperial: "20 - 22",
      height: "58 - 64",
      id: 10,
      name: "Alaskan ",
      group: "Working",
      lifespan: "12 - 15 years",
      temperament: "Friendly, Affectionate, Devoted, Loyal, Dignified, Playful",
    },
  ],
};
test("listHeader snapshot", () => {
  const element = new TestRenderer.create(
    (
      <DogDataProvider value={data}>
        <DogSearch />
      </DogDataProvider>
    )
  );
  expect(element).toMatchSnapshot();
});
