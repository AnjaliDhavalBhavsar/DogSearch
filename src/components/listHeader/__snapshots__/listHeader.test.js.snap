// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listHeader snapshot 1`] = `
Array [
  <div
    className="searchContainer"
  >
    <div
      className="searchText"
    >
      Search
    </div>
    <input
      aria-label="SearchInput"
      className="searchInput"
      onChange={[Function]}
      placeholder="Search for dog "
      type="text"
    />
  </div>,
  <div
    className="container"
  >
    <div>
      <div>
        <button
          className="headerButton"
          onClick={[Function]}
          style={
            Object {
              "backgroundImage": undefined,
              "width": "10%",
            }
          }
        >
          Image
        </button>
        <button
          className="headerButton"
          onClick={[Function]}
          style={
            Object {
              "backgroundImage": "url(iconsUpDown.png)",
              "width": "22.5%",
            }
          }
        >
          Name
        </button>
        <button
          className="headerButton"
          onClick={[Function]}
          style={
            Object {
              "backgroundImage": "url(iconsUpDown.png)",
              "width": "22.5%",
            }
          }
        >
          Height
        </button>
        <button
          className="headerButton"
          onClick={[Function]}
          style={
            Object {
              "backgroundImage": "url(iconsUpDown.png)",
              "width": "22.5%",
            }
          }
        >
          LifeSpan
        </button>
        <button
          className="headerButton"
          onClick={[Function]}
          style={
            Object {
              "backgroundImage": false,
              "width": "22.5%",
            }
          }
        >
          Breed Group
        </button>
      </div>
      <div
        className="containerNoData"
      >
        <img
          alt="No Data"
          height="400"
          src="noData.png"
          width="400"
        />
        <div>
          No Data Available
        </div>
      </div>
    </div>
  </div>,
]
`;
